Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN mkdir -p src \
     && cargo build --release --locked
```

Comment:
Do we want to do the same thing here, for dependencies?

Reply:
Ah, probably I don't understand how this works, ignore this.

Reply:
:/ oh yes, completely missed that.